samples/bluetooth/peripheral_hids_keyboard/src/main.c

```diff
@@ -752,8 +828,53 @@ static void button_shift_changed(bool down)
 	}
 }
 
+
+static void num_comp_reply(bool accept)
+{
+	struct pairing_data_mitm pairing_data;
+	struct bt_conn *conn;
+
+	if (k_msgq_get(&mitm_queue, &pairing_data, K_NO_WAIT) != 0) {
+		return;
+	}
+
+	conn = pairing_data.conn;
+
+	if (accept) {
+		bt_conn_auth_passkey_confirm(conn);
+		printk("Numeric Match, conn %p\n", conn);
+	} else {
+		bt_conn_auth_cancel(conn);
+		printk("Numeric Reject, conn %p\n", conn);
+	}
+
+	bt_conn_unref(pairing_data);
```

Comment:
bt_conn_unref(conn);

Reply:
pairing_data.conn or I need a second ref for conn ?

Reply:
`conn = pairing_data.conn` earlier, so it's the same.

Reply:
Ok, I fixed this.